src/sentry/models/externalissue.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BoundedPositiveIntegerField, Model, sane_repr
+
+
+class ExternalIssue(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField()
+    integration_id = BoundedPositiveIntegerField()
+    key = models.CharField(max_length=64)  # example APP-123 in jira
```

Comment:
Do we forsee needing to potentially bump this in the future? I don't know what all the options here are, but I feel like I'd rather just make this much wider to avoid it being an issue and not needing to ALTER it later.